### DISCUSSION TITLE
Saving settings redirects to /fetch/update-check

### DISCUSSION THREAD

**Discussion by AndiLeni** [Kovah/LinkAce#739] [Support]
Hello,

I tried hosting linkace on my own homeserver without docker. 
But there seems to be a small misconfiguration somewhere.

When I click "save settings" on the /settings page, I get redirected to /fetch/update-check which shows `{"checkResult":true}`.

Are there any tips on how I might solve this issue? 

Kind reagrds
Andreas

**Comment by Kovah** [comment]
I remember that long ago a user posted the same issue and narrowed down to a browser extension messing up with websites. But I can't find the thread anymore.

**Reply by brantg** [reply]
I get this same behavior all over the place.  Delete a tag, forward to update-check.  Delete a link, off to update-check.

I have three extensions and disabled them all, but this keeps happening.  I'm open to any suggestions - it works great otherwise.

**Reply by Kovah** [reply]
Are you seeing any errors in your browser console when trying to delete or open any page at all? Is your browser blocking any Javascript?

**Reply by brantg** [reply]
I can reliably reproduce it by clicking search on an empty query, and when I do that I just get this:

![image](https://github.com/Kovah/LinkAce/assets/10169603/e255b282-81d4-4134-b236-b10354ed61ed)

**Reply by Kovah** [reply]
Hm. I have no idea how to replicate this. Could you share more details about the LinkAce version and your browser?

**Reply by brantg** [reply]
Sure, that's on Firefox 125.0.3 (Flatpak), the LinkAce is v1.14.2.

I can also reproduce this the same way in Firefox v126 on Android and also on Microsoft Edge v125.0.2535.51 on Windows.

**Reply by Kovah** [reply]
Honestly, I have no idea why this happens and how to reproduce it to finde the error. It never happened to me before and there's no clue about what might cause it. I will have another look at the update check feature as soon as I can, but I can't promise anything. Sorry.

**Reply by brantg** [reply]
No worries!  Thank you for looking into this and if there is anything else I can do to assist you, just let me know.  I like LinkAce, so I'll still use it and just work around the errors.
